tests/doc-links.spec.ts

```diff
@@ -0,0 +1,218 @@
+import { test } from "./fixtures/lxd-test";
+import {
+  createInstance,
+  deleteInstance,
+  editInstance,
+  randomInstanceName,
+} from "./helpers/instances";
+import {
+  createProject,
+  deleteProject,
+  randomProjectName,
+} from "./helpers/projects";
+import { validateDocumentationLink } from "./helpers/doc-links";
+import { visitServerSettings } from "./helpers/server";
+
+test("Ensure the documentation link text and link targets are present: Server Settings > Override", async ({
+  page,
+}) => {
+  await visitServerSettings(page);
+
+  await validateDocumentationLink(
+    page,
+    "Separate REST API and clustering networks",
+    "/documentation/howto/cluster_config_networks/#cluster-https-address",
+    "cluster.https_address",
+  );
+  await validateDocumentationLink(
+    page,
+    "How to configure LXD as a BGP server",
+    "/documentation/howto/network_bgp/#network-bgp",
+    "core.bgp_address",
+  );
+  await validateDocumentationLink(
+    page,
+    "Enable the built-in DNS server",
+    "/documentation/howto/network_zones/#network-dns-server",
+    "core.dns_address",
+  );
+  await validateDocumentationLink(
+    page,
+    "How to expose LXD to the Network",
+    "/documentation/howto/server_expose/#server-expose",
+    "core.https_address",
+  );
+  await validateDocumentationLink(
+    page,
+    "How to monitor metrics",
+    "/documentation/metrics/",
+    "core.metrics_address",
+  );
+  await validateDocumentationLink(
+    page,
+    "How to manage storage buckets and keys",
+    "/documentation/howto/storage_buckets/#howto-storage-buckets",
+    "core.storage_buckets_address",
+  );
+  await validateDocumentationLink(
+    page,
+    "Instance placement scriptlet",
+    "/documentation/explanation/clustering/#clustering-instance-placement-scriptlet",
+    "instances.placement.scriptlet",
+  );
+});
+
+test("Ensure the documentation link text and link targets are present: Instance Detail > Configuration", async ({
+  page,
+}) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  await editInstance(page, instance);
+
+  await page.getByText("Security policies").click();
+  let buttons = page.getByRole("row").getByRole("button");
+  for (let i = 0; i < (await buttons.count()) - 1; i++) {
+    await buttons.nth(i).click();
+  }
```

Comment:
There is an `activateOverride` helper that might be useful instead of clicking all row buttons.